game-cards/SGgame.js

```diff
@@ -1201,9 +1201,91 @@ exports.commands = {
 		if (!p.bag.pokeballs[target[1]]) p.bag.pokeballs[target[1]] = 0;
 		p.bag.pokeballs[target[1]] += target[2];
 		Db.players.set(u.userid, p);
-		return this.sendReply(`${u.userid} has been given ${target[2]} ${target[1]}'s. They now have ${p.bag.pokeballs[target[1]]} ${target[1]}'s.`);
+		if (Rooms.get('staff')) {
+			Rooms.get('staff').add('|raw| <div class="broadcast-green">' + WL.name(user.name, true) + ' has given ' + target[2] + ' ' + target[1] + 's to ' + (u.userid) + '</div>').update();
+		}
```

Comment:
We dont really need to notify the staff room for this.

Reply:
It was to pervent other staff members to give themself unlimited pokeballs.

Reply:
Im not worried about my staff abusing this, feel free to just modlog it instead.